### PR TITLE
[CFE-553]: Feat(utils): move verify module api

### PR DESCRIPTION
**Pull request by Poafs1**

### CODE REVIEW

File: src/lib/services/types/verification/move/index.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    verified_at: z.string(),
+    digest: z.string(),
+    source: z.string(),
+    base64: z.string(),
+    chain_id: z.string(),
+  })
+  .transform(snakeToCamel);
+export type MoveVerifyInfoResponse = z.infer<typeof zMoveVerifyInfoResponse>;
+
+export const zMoveVerifyInfosByModuleResponse = z.object({
```
Comment (songwongtp) [inline src/lib/services/types/verification/move/index.ts:54]: ditto

File: src/lib/services/types/verification/move/index.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  contracts: z.array(
+    zMoveVerifyInfoResponse.innerType().extend({
+      id: z.string(),
+    })
+  ),
+});
+export type MoveVerifyInfosByModuleResponse = z.infer<
```
Comment (songwongtp) [inline src/lib/services/types/verification/move/index.ts:61]: ditto

File: src/lib/services/verification/move/api.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  address: Addr,
+  moduleName: string
+): Promise<Nullable<MoveVerifyInfoResponse>> =>
+  axios
+    .get(
+      `${INITIA_MOVE_VERIFIER}/contracts/verify/${encodeURI(address)}/${encodeURI(moduleName)}`
+    )
+    .then(({ data }) => parseWithError(zMoveVerifyInfoResponse, data));
+
+export const getMoveVerifyInfosByModule = async (
```
Comment (songwongtp) [inline src/lib/services/verification/move/api.ts:47]: ditto

File: src/lib/services/verification/move/index.ts [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    {
+      enabled: layer === "1" && Boolean(address && moduleName),
+      retry: 0,
+      refetchOnWindowFocus: false,
+      keepPreviousData: true,
+    }
+  );
+};
+
+export const useMoveVerifyInfosByModule = (
```
Comment (songwongtp) [inline src/lib/services/verification/move/index.ts:84]: ditto

**Review by songwongtp — CHANGES_REQUESTED** [review]

**Review by songwongtp — APPROVED** [review]

File: src/lib/services/verification/move/index.ts [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+export const useSubmitMoveVerify = (
+  formData: FormData
+): UseQueryResult<SubmitMoveVerifyResponse> => {
+  const { chainConfig } = useCelatoneApp();
+  const {
+    extra: { layer },
+  } = chainConfig;
+
+  return useQuery(
```
Comment (evilpeach) [inline src/lib/services/verification/move/index.ts:28]: Please use `useMutation`.

File: src/lib/services/verification/move/index.ts [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+    [CELATONE_QUERY_KEYS.MOVE_VERIFY_SUBMIT, formData, layer],
+    () => getSubmitMoveVerify(formData),
```
Comment (evilpeach) [inline src/lib/services/verification/move/index.ts:30]: Don't name get function because it's post function.

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**